docs/dev/development-setup.rst

```diff
@@ -238,9 +238,8 @@ Open yet another terminal for client and run:
 
 .. code-block:: console
 
-   $ export BACKEND_ENDPOINT=http://127.0.0.1:8081/  # change the port number if customized
-   $ export BACKEND_ACCESS_KEY=AKIAIOSFODNN7EXAMPLE
-   $ export BACKEND_SECRET_KEY=wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
+   $ source ./env-local-admin-api.sh # generated in project root if install-dev.sh worked correctly.
+   $ # source ./env-local-user-api.sh # Choose and use the environment you want.
```

Comment:
Changes made here are identical to the contents [this PR](https://github.com/lablup/backend.ai/pull/1463) covers. Could you remove these lines?